### PR TITLE
Add WebRTC Redirector Service

### PR DESCRIPTION
**Pull request by alexwilcox9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

Adding the WebRTC Redirector Service for Teams on AVD, more information here:
https://docs.microsoft.com/en-us/azure/virtual-desktop/teams-on-avd

Note the `InstallerUrl` is a generic URL for the latest version as I couldn't find a URL for this specific version

-----

This is a follow up to https://github.com/microsoft/winget-pkgs/pull/38391 and now has the package dependencies called out in the YAML file

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/45574)

### CODE REVIEW

File: manifests/m/Microsoft/WebRTCRedirectorService/1.4.2111.18001/Microsoft.WebRTCRedirectorService.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.WebRTCRedirectorService
+PackageVersion: 1.4.2111.18001
+PackageLocale: en-US
+Publisher: Microsoft
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/WebRTCRedirectorService/1.4.2111.18001/Microsoft.WebRTCRedirectorService.locale.en-US.yaml:7]:
```suggestion
Publisher: Microsoft Corporation
```

File: manifests/m/Microsoft/WebRTCRedirectorService/1.4.2111.18001/Microsoft.WebRTCRedirectorService.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.WebRTCRedirectorService
+PackageVersion: 1.4.2111.18001
+Installers:
+  - InstallerLocale: en-US
+    Architecture: x64
+    InstallerType: msi
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/WebRTCRedirectorService/1.4.2111.18001/Microsoft.WebRTCRedirectorService.installer.yaml:9]:
```suggestion
    InstallerType: wix
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Please set the PR as a Draft until Dependencies are out of experimental
```
Enter a pull request number: #45574

Manifest has the following dependencies that were not validated; ensure that they are valid:
  - Packages
      Microsoft.VC++2015-2022Redist-x86
      Microsoft.VC++2015-2022Redist-x64
Manifest validation succeeded.
Found Remote Desktop WebRTC Redirector Service [Microsoft.WebRTCRedirectorService] Version 1.4.2111.18001
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
This package requires the following dependencies:
  - Packages
      Microsoft.VC++2015-2022Redist-x86
      Microsoft.VC++2015-2022Redist-x64
Installing dependencies:
(1/2) Found Microsoft Visual C++ 2015-2022 Redistributable (x64) [Microsoft.VC++2015-2022Redist-x64] Version 14.31.31103.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.visualstudio.microsoft.com/download/pr/d22ecb93-6eab-4ce1-89f3-97a816c55f04/37ED59A66699C0E5A7EBEEF7352D7C1C2ED5EDE7212950A1B0A8EE289AF4A95B/VC_redist.x64.exe
  ██████████████████████████████  24.1 MB / 24.1 MB
Successfully verified installer hash
Starting package install...
Successfully installed

(2/2) Found Microsoft Visual C++ 2015-2022 Redistributable (x86) [Microsoft.VC++2015-2022Redist-x86] Version 14.31.31103.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.visualstudio.microsoft.com/download/pr/8e32d7eb-5130-4dc8-9c3e-5891f375e112/B7AE307237F869E09F7413691A2CD1944357B5CEE28049C0A0D3430B47BB3EDC/VC_redist.x86.exe
  ██████████████████████████████  13.0 MB / 13.0 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Successfully verified installer hash
Starting package install...
Successfully installed

Name        : Microsoft Visual C++ 2015-2022 Redistributable (x64) - 14.31.31103
Publisher   : Microsoft Corporation
Version     : 14.31.31103.0
ProductCode : {2aaf1df0-eb13-4099-9992-962bb4e596d1}
Uninstall   : winget uninstall --id "{2aaf1df0-eb13-4099-9992-962bb4e596d1}"

Name        : Microsoft Visual C++ 2015-2022 Redistributable (x86) - 14.31.31103
Publisher   : Microsoft Corporation
Version     : 14.31.31103.0
ProductCode : {41d7b770-418a-43b7-95a5-f925fff05789}
Uninstall   : winget uninstall --id "{41d7b770-418a-43b7-95a5-f925fff05789}"

Name        : Remote Desktop WebRTC Redirector Service
Publisher   : Microsoft Corporation
Version     : 1.4.2111.18001
ProductCode : {729B4FCB-E748-4ED0-978D-72512B332017}
Uninstall   : winget uninstall --id "{729B4FCB-E748-4ED0-978D-72512B332017}"
```

**Comment by alexwilcox9** [comment]
PR is now in draft and the suggested changes have been made
Thanks

**Comment by OfficialEsco** [comment]
@denelon can we get a dependency label? 🙃

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**